community-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
```

Comment:
> It is specifically a polyglot group concerned to discuss programming ideas over the long-term.

Maybe: It is a polyglot group for friendly discussion around programming & technology ideas over the long-term.